cluster/init.sls

```diff
@@ -26,6 +26,9 @@ include:
 {% endif %}
 {% if cluster.corosync is defined %}
   - .corosync
+{% if cluster.init == host %}
```

Comment:
Is intentional to have this within the `if cluster.corosync is defined` condition?